extensions/azurePublish/src/components/ChooseProvisionAction.tsx

```diff
@@ -0,0 +1,264 @@
+    </Content>
+  );
+};
+
+const ImportActionContent = () => {
+  return (
+    <Content>
+      <Title>{formatMessage('Import existing resources')}</Title>
+      <Summary>
+        <p>
+          <Text>{formatMessage('Select this option to import existing Azure resources and publish a bot.')}</Text>
+        </p>
+        <p>
+          <Text>
+            {formatMessage(
+              'Edit the JSON file in the Publish Configuration field. You will need to find the values of associated resources in your Azure portal. A list of required and optional resources may include:'
+            )}
+          </Text>
+        </p>
+      </Summary>
+      <Details>
+        <ResourceTitle>
+          <Text>Microsoft Application Registration</Text>
+        </ResourceTitle>
+        <ResourceTitle>
+          <Text>Azure Hosting</Text>
+        </ResourceTitle>
+        <ResourceTitle>
+          <Text>Microsoft Bot Channels Registration </Text>
+        </ResourceTitle>
+        <ResourceTitle>
+          <Text>Azure Cosmos DB </Text>
+        </ResourceTitle>
+        <ResourceTitle>
+          <Text>Application Insights </Text>
+        </ResourceTitle>
+        <ResourceTitle>
+          <Text>Azure Blob Storage </Text>
+        </ResourceTitle>
+        <ResourceTitle>
+          <Text>Microsoft Language Understanding (LUIS)</Text>
+        </ResourceTitle>
+        <ResourceTitle>
+          <Text>Microsoft QnA Maker </Text>
+        </ResourceTitle>
+      </Details>
+      <LearnMoreLink>{formatMessage('Learn More')}</LearnMoreLink>
+    </Content>
+  );
+};
+
+const GenerateActionContent = () => {
+  return (
+    <Content>
+      <Title>{formatMessage('Hand off to admin')}</Title>
+      <Summary>
+        <Text>
+          {formatMessage(
+            'Select this option to request your Azure admin to provision resources on your behalf, for example, when you don’t have proper permissions to use Azure or you want to generate resources from a sovereign cloud.'
+          )}
+        </Text>
+      </Summary>
+      <Details>
+        <Instruction>
+          <InstructionTitle>{formatMessage('Step 1')}</InstructionTitle>
+          <InstructionDetails>
+            <Text>
+              {formatMessage(
+                'Add resources you need for the bot and generate a resource request to share with your Azure admin.'
+              )}
+            </Text>
+          </InstructionDetails>
+        </Instruction>
+        <Instruction>
+          <InstructionTitle>{formatMessage('Step 2')}</InstructionTitle>
+          <InstructionDetails>
+            <Text>
+              {formatMessage(
+                'Once you get the resource details from your Azure admin, use them to import existing resources.'
+              )}
+            </Text>
+          </InstructionDetails>
+        </Instruction>
+      </Details>
+      <LearnMoreLink>{formatMessage('Learn More')}</LearnMoreLink>
```

Comment:
needs href

Reply:
I will add when design/docs has one. No idea what to put here.